### PR TITLE
removed void scans and helvetica scans

### PR DESCRIPTION
**Pull request by OncePunchedMan**

### CODE REVIEW

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/foolslide/FoolSlideGenerator.kt [hunk 1]
```diff
@@ -34,7 +34,6 @@ class FoolSlideGenerator : ThemeSourceGenerator {
         SingleLang("Storm in Heaven", "https://www.storm-in-heaven.net", "it", className = "StormInHeaven"),
         SingleLang("LupiTeam", "https://lupiteam.net", "it"),
         SingleLang("Zandy no Fansub", "https://zandynofansub.aishiteru.org", "en"),
-        SingleLang("Helvetica Scans", "https://helveticascans.com", "en"),
```
Comment (arkon) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/foolslide/FoolSlideGenerator.kt:37]: Remove the override files too.

**Review by arkon — CHANGES_REQUESTED** [review]

**Comment by jopejoe1** [comment]
Did i remove wrong void scan in #6421 ?

**Comment by arkon** [comment]
> Did i remove wrong void scan in #6421 ?

@CrepeTF would need to confirm that since the removal issue didn't specify. Looks like this site is dead anyway though.

**Comment by OncePunchedMan** [comment]
hm both voidscans and ntsvoidscans are non-reachable

**Comment by CrepeTF** [comment]
> > Did i remove wrong void scan in #6421 ?
> 
> @CrepeTF would need to confirm that since the removal issue didn't specify. Looks like this site is dead anyway though.

Yep it's correct @arkon

**Pull request merged**